skyportal/models/galaxy.py

```diff
@@ -0,0 +1,23 @@
+__all__ = ['Galaxy']
+
+import sqlalchemy as sa
+import healpix_alchemy as ha
+
+from baselayer.app.models import Base
+
+
+class Galaxy(Base, ha.Point):
+    """A record of a galaxy and its metadata, such as position,
```

Comment:
Should probably have an "alt_name" list field here, since there may be many names for the galaxy.

Would we ever have the case that a `Galaxy` would need user annotations or other `altdata` fields for storing generic data?

Reply:
Some catalogs may have redshift and redshift uncertainty, distance uncertainty, etc.

Reply:
I went ahead and added alt_name, redshift and redshift uncertainty, and distance uncertainty. Maybe the group can discuss if they want the altdata bit.